doc/tor.md

```diff
@@ -57,11 +57,7 @@ outgoing connections, but more is possible.
     -onlynet=onion  Make outgoing connections only to .onion addresses. Incoming
                     connections are not affected by this option. This option can be
                     specified multiple times to allow multiple network types, e.g.
-                    onlynet=ipv4, onlynet=ipv6, onlynet=onion, onlynet=i2p.
-                    Warning: if you use -onlynet with values other than onion, and
-                    the -onion or -proxy option is set, then outgoing onion
-                    connections will still be made; use -noonion or -onion=0 to
-                    disable outbound onion connections in this case.
+                    onlynet=ipv6, onlynet=onion.
```

Comment:
(Initial drive-by review.) Every onlynet question I've received has been about using -onlynet=onion, -onlynet=i2p, or both together, so it may be reasonable to address what node operators seem most interested in and minimize confusion, by using the hidden networks for the onlynet examples in both `doc/tor.md` and `doc/i2p.md` (unless you use all of them in the examples to illustrate that it is possible, which is what I did in #22648, but no strong opinion on whether that is helpful)
```suggestion
                    onlynet=onion, onlynet=i2p.
```

Reply:
Ok, I will update this as per above. I changed it because it was using onlynet with all possible networks which looked strange to me.

Reply:
Done.